src/test/resources/pages/instructorFeedbackSubmitPageRankHelper.html

```diff
@@ -121,10 +121,10 @@
             <div class="col-sm-9">
               <div>
                 <p class="text-muted" hidden="">
-                  You need to rank at least  options.
+                  You need to rank at least  ${optionsOrRecipients}.
```

Comment:
Not a big deal from user's POV since this is hidden, but from a tester's POV `${...}` is most of the time associated with God mode. See if you can still inject some appropriate values here.

Reply:
I had a similar feeling, will look into it.